apps/scan/backend/src/electrical_testing/tasks/print_and_scan_task.ts

```diff
@@ -70,30 +74,42 @@ export async function runPrintAndScanTask({
         `Scanned sheet: front=${front.width}×${front.height}, back=${back.width}×${back.height}`
       );
 
-      const usbDriveStatus = await usbDrive.status();
-
-      const basedir =
-        usbDriveStatus.status === 'mounted'
-          ? join(usbDriveStatus.mountPoint, 'ballot-images')
-          : workspace.ballotImagesPath;
-      const basename = `electrical-testing-${lastScanTime
+      const sheetId = `electrical-testing-${lastScanTime
         .toISO()
         .replaceAll(':', '-')}`;
-
-      await mkdir(basedir, { recursive: true });
       await saveSheetImages({
-        sheetId: basename,
-        ballotImagesPath: basedir,
+        sheetId,
+        ballotImagesPath: workspace.ballotImagesPath,
         images: scannerEvent.images,
       });
 
-      await writeScanPageAnalyses(
-        logger,
-        lastScanTime,
-        [front, back],
-        basedir,
-        basename
-      );
+      if (
+        isFeatureFlagEnabled(
+          BooleanEnvironmentVariableName.ENABLE_HARDWARE_TEST_APP_INTERNAL_FUNCTIONS
+        )
+      ) {
+        const usbDriveStatus = await usbDrive.status();
+        if (usbDriveStatus.status === 'mounted') {
+          const usbDriveExportDirectory = join(
+            usbDriveStatus.mountPoint,
+            'ballot-images'
+          );
+          await mkdir(usbDriveExportDirectory, { recursive: true });
+
+          await saveSheetImages({
+            sheetId,
+            ballotImagesPath: usbDriveExportDirectory,
+            images: scannerEvent.images,
+          });
```

Comment:
I recognize that, for internal use, we're now both writing to disk and the USB drive if connected. I think that's fine. Made the code and functionality a little bit easier to reason about when set up that way